src/Core/Components/Slider/FluentSlider.razor.cs

```diff
@@ -41,60 +37,43 @@ public FluentSlider()
         }
     }
 
-    /// <summary>
-    /// Represents a reference to a DOM element in the component. It allows interaction with the specified element in
-    /// the browser.
-    /// </summary>
-    [Parameter]
-    public ElementReference Element { get; set; }
-
     /// <summary>
     /// Gets or sets the size for the slider.
     /// </summary>
     [Parameter]
     public SliderSize? Size { get; set; }
 
     /// <summary>
-    /// Gets or sets the slider's minimal value.
+    /// Gets or sets the slider's minimal value. Default is 0.
     /// </summary>
     [Parameter]
     public TValue? Min { get; set; }
 
     /// <summary>
-    /// Gets or sets the slider's maximum value.
+    /// Gets or sets the slider's maximum value. Default is 100.
     /// </summary>
     [Parameter]
     public TValue? Max { get; set; }
 
     /// <summary>
-    /// Gets or sets the slider's step value.
+    /// Gets or sets the slider's step value. Default is 1.
     /// </summary>
     [Parameter]
     public TValue? Step { get; set; }
 
     /// <summary>
-    /// Gets or sets the orientation of the stacked components.
+    /// Gets or sets the orientation of the stacked components. Default is <see cref="Orientation.Horizontal"/>.
```

Comment:
/// Gets or sets the orientation of the component. Default is <see cref="Orientation.Horizontal"/>.